crates/ika-core/src/dwallet_mpc/crytographic_computation/mpc_computations/network_dkg.rs

```diff
@@ -276,11 +281,12 @@ impl DwalletMPCNetworkKeys {
         })
     }
 
-    /// Retrieves the decryption key shares for the current authority.
-    pub(crate) fn get_decryption_key_shares(
+    /// Retrieves the secp256k1 decryption key shares for the current authority.
+    pub(crate) fn secp256k1_decryption_key_shares(
```

Comment:
```suggestion
    pub(crate) fn decryption_key_shares(
```

its the same for all curves.